ui/raidboss/browser_tts_engine.ts

```diff
@@ -56,7 +57,7 @@ export default class BrowserTTSEngine {
 
     // figure out what TTS engine type we need
     const speechLang = cactbotLangToSpeechLang[this.cactbotLang];
-    const voice = window.speechSynthesis.getVoices().find((voice) => voice.lang === speechLang);
+    const voice = window.speechSynthesis.getVoices().find((voice) => voice.lang.replaceAll('_','-') === speechLang);
```

Comment:
```suggestion
    const voice = window.speechSynthesis.getVoices().find((voice) => {
      return voice.lang === speechLang || voice.lang.replaceAll('_', '-') === speechLang);
    });
```

I think this will break desktop if you always replace.  Better to check both the hyphen and the underscore versions so we can support both.